### PR TITLE
Add linger status to `loginctl list-users`

### PR DESCRIPTION
**Pull request by dtardon**

### CODE REVIEW

**Comment by yuwata** [comment]
LGTM. Could you add a test case?

**Comment by dtardon** [comment]
> LGTM. Could you add a test case?

Done.

File: test/units/testsuite-35.sh [hunk 1]
```diff
@@ -445,6 +447,25 @@ test_session_properties() {
     /usr/lib/systemd/tests/manual/test-session-properties "/org/freedesktop/login1/session/_3${s?}"
 }
 
+test_list_users() {
+    local u
```
Comment (yuwata) [inline test/units/testsuite-35.sh]:
```
In /github/workspace/test/units/testsuite-35.sh line 451:
    local u
          ^-- SC2034 (warning): u appears unused. Verify use (or export if used externally).
```

**Review by yuwata — APPROVED** [review]
LGTM. Except for the warning by the shell check.

**Pull request merged**